### PR TITLE
Add document to join Mattermost

### PR DESCRIPTION
**Pull request by Aysha-py**
The link to the images in the document is currently broken. However, it is available on the cloud. Can someone please help fix this or provide assistance?

You can find the images [here](https://cloud.opentermsarchive.org/index.php/apps/files/?dir=/Workspace&fileid=1390)

### CODE REVIEW

**Comment by clementbiron** [comment]
Hello @Aysha-py, 

Thanks for your work!

I'll take care of correcting the links to the images but the url you provide won't let me retrieve them. Can you check that the link and/or share configuration is correct? If need be, let's call to discuss it together.

**Comment by MattiSG** [comment]
Here is the [direct file link](https://cloud.opentermsarchive.org/index.php/apps/files/?dir=/Workspace&openfile=1466).

**Comment by clementbiron** [comment]
Thanks @MattiSG

**Comment by Aysha-py** [comment]
Thanks @MattiSG for providing the file link and @clementbiron for adding the images.

**Review by clementbiron — COMMENTED** [review]
Thanks for your work! 

I've made the necessary corrections to get the images working.

I know @MattiSG will be doing a review for you in the next few days.

File: content/community/how-to-join-mattermost.md [hunk 1]
```diff
@@ -3,3 +3,67 @@ title: How to join Mattermost
...
+
+The Open Terms Archive workspace consists of several channels. Some are specific while others are general. New members and contributors can add any channel of your choice by following these steps:
+
+1. Navigate to the "**CHANNELS**" section on the left-hand sidebar.
+2. Toggle the "**CHANNELS**" button and click on “**Add Channels**”.
+   ![Adding Channels on Mattermost]({{< relref "." >}}adding-channels.png)
+3. A drop-down menu appears. Click “**Browse channel**” to see the list of all available channels on the Open Terms Archive workspace.
+   ![See Channels on Mattermost]({{< relref "." >}}see-channels.jpg)
+4. Find a channel that interests you from the list of available channels, and click **”join”** to add it to your workspace.
+   ![Joinning Channels on Mattermost]({{< relref "." >}}joinning-channels.png)
```
Comment (MattiSG) [inline content/community/how-to-join-mattermost.md:47]:
```suggestion
   ![Joining Channels on Mattermost]({{< relref "." >}}joinning-channels.png)
```

**Review by MattiSG — CHANGES_REQUESTED** [review]
Looks good, thank you! 👍 
I made some suggestions to correct a few typos and to unify the markup.

**Review by MattiSG — APPROVED** [review]

**Pull request merged**